DataConnectors/AWS-S3/CloudWatchLanbdaFunction.py

```diff
@@ -0,0 +1,59 @@
+
+import boto3
+import json
+import csv
+import time
+import pandas as pd
+from datetime import datetime
+
+logs = boto3.client('logs')
+s3 = boto3.resource('s3')
+
+
+# Please set the following parameters:
```

Comment:
All should be on env vars

Reply:
I don't agree. I don't have a way to share the lamda itself but only the source code. This means I have to give instructions to the customers how to define those variables. The instructions already contain 24 steps, I really wouldn't like to add more